src/www/home.html

```diff
@@ -54,58 +54,83 @@
-                <div class='col-md-4 flex flex-around flex-center text-center'>
+    </div>
+</section>
 
-                    <div>
-                        <span class='text-primary info_graphic'>130</span>
-                        <p>Honors students</p>
-                    </div>
-                    <div>
-                        <span class='text-primary info_graphic'>190</span>
-                        <p>Alumni</p>
-                    </div>
-                    <div>
-                        <span class='text-primary info_graphic'>8</span>
-                        <p>Committees</p>
-                    </div>
-                    <div>
-                        <span class='text-primary info_graphic'>8</span>
-                        <p>Tracks</p>
-                    </div>
-                </div>
+<section class="block container">
+    <div class="row">
+
+        <div class='col-sm-12 text-center' >
+
+            <div class="block col-xs-6 col-md-3">
+                <span class='text-primary info_graphic'>130</span>
+                <p>Honors students</p>
+            </div>
+            <div class="block col-xs-6 col-md-3">
+                <span class='text-primary info_graphic'>190</span>
+                <p>Alumni</p>
+            </div>
+            <div class="block col-xs-6 col-md-3">
+                <span class='text-primary info_graphic'>8</span>
+                <p>Committees</p>
+            </div>
+            <div class="block col-xs-6 col-md-3">
+                <span class='text-primary info_graphic'>8</span>
+                <p>Tracks</p>
             </div>
 
         </div>
 
+    </div>
 </section>
-<section class="section container">
-    <div class="block row small">
-
-        <div class="col-xs-12 col-md-6 center text-center">
-            <p class="h3">Connect</p>
-            <p><i>H.S.A. Confluente aims to increase both the professional and social network of honors students.</i>
-            </p>
-            <p> Members are welcome to attend our various activities such as lunch lectures, workshops, drinks, study
-                trips and more. In addition, the association builds upon a strong <a href="/about/alumni">Alumni
-                    network</a> and has close ties with business and entrepreneurial things.
-            </p>
+
+<section class="block bg-info">
+    <div class="container home-section animation-element">
+        <div class="row">
+
+            <div class="col-xs-12 col-md-6">
+                <p class="h3">Connect</p>
+                <p><i>H.S.A. Confluente aims to increase both the professional and social network of honors students.</i></p>
+                <p> Members are welcome to attend our various activities such as lunch lectures, workshops, drinks, study
+                    trips and more. In addition, the association builds upon a strong <a href="/about/alumni">Alumni
+                        network</a> and has close ties with business and entrepreneurial things.
```

Comment:
I suggest making this "and has close ties with different entrepreneurs, many student teams and corporations!" Or something along those lines